src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -0,0 +1,52 @@
+package lotto.domain;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class LottoNumber {
+
+    public static final int MIN_LOTTO_NUMBER = 1;
+    public static final int MAX_LOTTO_NUMBER = 45;
+
+    final private int number;
+
+    public LottoNumber(int number) {
+        if (!isValidLottoNumber(number)) {
+            throw new IllegalArgumentException("유효하지 않은 로또 번호 입니다.");
+        }
+
+        this.number = number;
+    }
+
+    public static List<LottoNumber> createLottoNumbers(int...args) {
+        return Arrays.stream(args).mapToObj(LottoNumber::new).collect(Collectors.toList());
```

Comment:
```suggestion
        return Arrays.stream(args)
			.mapToObj(LottoNumber::new)
			.collect(Collectors.toList());
```

한 라인에는 온점(.)을 하나만 적어보면 어떨까요?